tsdb/internal/meta.pb.go

```diff
@@ -16,7 +16,7 @@ It has these top-level messages:
 */
 package internal
 
-import proto "github.com/golang/protobuf/proto"
```

Comment:
Actually, it looks like you are changing it the correct import `gogo`, but it seems like this change is unrelated to the shard mapping code.